iguana/iguana_notary.c

```diff
@@ -559,7 +559,7 @@ STRING_ARG(iguana,addnotary,ipaddr)
 }
 
 // TODO: Confirm that this is the correct list of currencies. Is TOKEL supposed to be in here? It's the only 3rd party coin in the list.
-char NOTARY_CURRENCIES[][65] = { "SUPERNET", "NINJA", "CCL", "PIRATE", "ILN", "DOC", "THC", "MARTY", "KOIN", "GLEEC", "TOKEL", "CLC" };
+char NOTARY_CURRENCIES[][65] = { "SUPERNET", "NINJA", "CCL", "PIRATE", "ILN", "DOC", "THC", "MARTY", "KOIN", "GLEEC", "TOKEL", "CLC", "VOTE2024" };
```

Comment:
It seems to me that the `NOTARY_CURRENCIES` array is used only in deprecated Iguana RPCs, such as `fundnotaries` or `notarychains`, and doesn't affect the actual notarization process. Therefore, we can likely leave it untouched. As we observed when we initially added the KIPxxxx chains, we didn't modify the `NOTARY_CURRENCIES` array, and notarizations proceeded without issues. So, in the future, we can try to leave this array untouched.